test/functionalTest/cases/1108_non_string_fields/non_string_fields.test

```diff
@@ -0,0 +1,194 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Looking closer to the .test, it seems that is not only a matter of entity::id and entity::type. Maybe my previous comment doesn't make sense, or the suggested name is not the right one.